/**
 * @file   gp_wind_regression_main.h
 * @author Mohit Mehndiratta
 * @date   Sep 2021
 *
 * @copyright
 * Copyright (C) 2021.
 */

/**
 * @brief
 */

#ifndef GP_MODEL_H
#define GP_MODEL_H

#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wpedantic"
// Bad headers with problem go here
#include <ros/ros.h>
#include <ros/console.h>
#include <ros/time.h>
// restore compiler switches
#pragma GCC diagnostic pop

/// External imports

/// Internal imports
#include <gp_wind_regression/launch_parameters.h>
#include <gp_wind_regression/data_struct.h>
#include <gp_wind_regression/utils.h>

namespace wind_regression
{


}  // end namespace wind_regression

#endif
